fiona/__init__.py

```diff
@@ -202,6 +202,16 @@ def open(fp, mode='r', driver=None, schema=None, crs=None, encoding=None,
     crs_wkt : str
         An optional WKT representation of a coordinate reference
         system.
+    ignore_fields : list
+        List of field names to ignore on load.
+    ignore_geometry : bool
+        Ignore the geometry on load.
+    include_fields : list
+        List of a subset of field names to include on load.
+    wkt_version : str
+        WKT Version to use to for the CRS
+        (WKT1, WKT1_GDAL, WKT1_ESRI, WKT2, WKT2_2015, WKT2_2018).
```

Comment:
for the Fiona API, I would suggest using WKT2_2019 which is the canonical name and remap that to WKT2_2018 when passing it to GDAL for compatibility with early GDAL 3.1 (WKT2_2019 was added in GDAL 3.2)